pkg/actuators/machine/utils.go

```diff
@@ -113,6 +113,20 @@ func getInstances(machine *clusterv1.Machine, client awsclient.Client, instanceS
 	return instances, nil
 }
 
+func getVolume(client awsclient.Client, volumeID string) (*ec2.Volume, error) {
```

Comment:
shouldn't this be using `volumeID` for the request??

Reply:
thanks